internal/ignition/disconnected_test.go

```diff
@@ -73,6 +74,7 @@ var _ = Describe("Disconnected Ignition", func() {
 		}
 
 		id := strfmt.UUID("test-infra-env-id")
+		clusterID := strfmt.UUID("test-cluster-id")
```

Comment:
With late binding, an infraenv will be bound to the cluster later. A cluster object will be created by below the sea level UI when a user will boot the ISO.